src/components/InclusaoDeAlimentacao/components/CardPendenteAcao/index.jsx

```diff
@@ -97,15 +97,15 @@ export class CardPendenteAcao extends Component {
               </thead>
               <tbody>
                 {pedidosFiltrados.map((pedido, key) => {
-                  const dataMaisProxima =
-                    pedido.inclusoes && pedido.inclusoes[0].data;
+                  const dataMaisProxima = pedido.inclusoes ? pedido.inclusoes[0].data : pedido.data
+                    const isCei = !!pedido.isCei
+                    const isContinua = !!pedido.data_inicial
```

Comment:
Porque as duas exclamações?

Reply:
Converts X to (non inverted) boolean. 
Cf: https://stackoverflow.com/a/784946

Reply:
Ah, entendi, pra aparecer bonitinho na URL. Blz!

Reply:
Vem da url como tipo string, no código precisa ser um booleano.